crates/host/src/wasmbus/mod.rs

```diff
@@ -3183,6 +3199,7 @@ impl Host {
             );
             let otel_config = OtelConfig {
                 traces_exporter: self.host_config.otel_config.traces_exporter.clone(),
+                metrics_exporter: self.host_config.otel_config.metrics_exporter.clone(),
```

Comment:
Note that providers don't know how to handle this (yet?) so this is the same as `metrics_exporter: None`

Reply:
I've nixed this.